### DISCUSSION TITLE
Error ocurred

### DISCUSSION THREAD

**Discussion by begovegar** [PaddlePaddle/PaddleSpatial#236] [BaiduKDDCup2022]
I got an error message as follows. May I see the full log, please? Our team name is Innova Team.  
Last two submissions.
![image](https://user-images.githubusercontent.com/104911705/177097024-30192458-08e5-4e21-a891-22421ed67282.png)

**Comment by begovegar** [comment]
We already fixed it, thanks.

**Comment by zhiyansong** [comment]
I have also encountered the same problem, the error only shows the exception, may I ask whether your problem has been solved?

**Comment by begovegar** [comment]
It was just a problem of one of test sets apparently with no nans to replace. Just add an if statetement.
